tests/src/test/scala/cats/tests/ChainSuite.scala

```diff
@@ -56,6 +56,12 @@ class ChainSuite extends CatsSuite {
     }
   }
 
+  test("headOption") {
```

Comment:
nitpick: this doesn't test the None scenario. How about 

```scala
forAll { (s: Seq[Int]) =>
      Chain.fromSeq(s).headOption should ===(s.headOption)
    }
```

Reply:
Addressed. Thanks.